content/devops/foundational-short-course/2-why-is-devops-important/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+content_type: topic
+ready: true
+title: Why is DevOps important?
+---
+
+Before DevOps, a major methodology of developing software was used: waterfall. It consists of breaking down the development process into sequential, linear tasks, executing one after another and never reviewing a task after it is completed. It flows steadily downwards, like a waterfall.
+
+It entails a long planning phase, with strict time schedule, budget, and implementation of the entire project in one shot. Also, it brings strict documentation, a very formal review process, and sign off from IT ops between every phase.
+
+In one word, it is **inflexible**. Project’s demands changes, team changes, budget changes, features changes, management changes. Everything is very dynamic (and unstable!) throughout the software development process and DevOps came as a different way to think about it.
```

Comment:
Project's demands change. Teams change, feature changes
Are the three grammatical improvements I'd consider making